insights/parsers/ip.py

```diff
@@ -144,6 +148,16 @@ def parse_tx_stats(line, d):
     d["tx_collsns"] = int(split_content[5])
 
 
+def parse_vxlan_details(line, d):
```

Comment:
The computing of the function is quite simple, better remove the function and move the line to the calling place.

Reply:
Done